miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/BoxRestController.java

```diff
@@ -99,6 +118,134 @@ protected BoxDto asDto(Box model) {
     return Dtos.asBoxDtos(results, true);
   }
 
+  /**
+   * Creates an Excel spreadsheet that contains the list of all Boxable items located in a particular position. Empty positions are not
+   * listed.
+   *
+   * @param boxId ID of the Box
+   * @return JSON object with "hashCode" field representing the hash code of the spreadsheet filename
+   */
+  @RequestMapping(value = "/box/{boxId}/spreadsheet", method = RequestMethod.GET)
+  public @ResponseBody JSONObject createSpreadsheet(@PathVariable("boxId") Long boxId) {
+    try {
+      return exportBoxContentsForm(boxId);
+    } catch (Exception e) {
+      throw new RestException("Failed to get contents form", Status.BAD_REQUEST);
+    }
+  }
+
+  private JSONObject exportBoxContentsForm(Long boxId) throws IOException  {
+    Box box = boxService.get(boxId);
+
+    List<List<String>> boxContents = getBoxContents(box);
+    String name = box.getName();
+    String alias = box.getAlias();
+
+    File f = misoFileManager.getNewFile(Box.class, "forms", "BoxContentsForm-" + getCurrentDateAsString() + ".xlsx");
+    if (detailedSample) {
+      FormUtils.createDetailedBoxSpreadsheet(f, name, alias, boxContents);
+    } else {
+      FormUtils.createPlainBoxSpreadsheet(f, name, alias, boxContents);
+    }
+
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)